tests/test_models_unet.py

```diff
@@ -114,7 +114,8 @@ def prepare_init_args_and_inputs_for_common(self):
             "in_channels": 4,
             "out_channels": 4,
             "layers_per_block": 2,
-            "block_out_channels": (32, 64),
+            "block_out_channels": (16, 32),
```

Comment:
Wait why are we changing this here? IMO this should stay exactly the same - otherwise we're loosing a test case that was there before

Reply:
my bad, leftover from previous commits